docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -12,34 +18,35 @@ To use the component, simply combine it with the rest of your UI, e.g. in `setCo
 ```kotlin
 override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    
+
     // Load the data for the header here
-    
+
     setContent {
-        Column(Modifier.fillMaxSize()) {
-            MessageListHeader(
-                channel,
-                currentUser,
-                connectionState,
-                messageMode,
-                modifier = Modifier.fillMaxWidth(),
-                onBackPressed = { },
-                onHeaderActionClick = { },
-            )
-
-            // Rest of your UI
+        ChatTheme {
+            Column(Modifier.fillMaxSize()) {
+                MessageListHeader(
+                    modifier = Modifier.wrapContentHeight(),
```

Comment:
Hm, isn't it already wrapping its height?

Reply:
Unfortunately the  `Column` inside `MessageListHeader`  used to use `fillMaxHeight()` which caused issues.
Now its using `height(IntrinsicSize.Max)` which allows for greater call site flexibility.